README.md

```diff
@@ -34,25 +34,58 @@ and another Kill Bill system operating for handling payments.
 The plugin will need to have the default configuration parameter to connect to the remote Kill Bill (payment) system.
 In addition for each tenant, the details of the `api_key` and `api_secret` will be required.
 
-Thw following properties are supported:
-
-* `org.killbill.billing.plugin.bridge.serverHost`
-* `org.killbill.billing.plugin.bridge.serverPort`
-* `org.killbill.billing.plugin.bridge.username`
-* `org.killbill.billing.plugin.bridge.password`
-* `org.killbill.billing.plugin.bridge.apiKey`
-* `org.killbill.billing.plugin.bridge.apiSecret`
-* `org.killbill.billing.plugin.bridge.proxyHost` (Optional)
-* `org.killbill.billing.plugin.bridge.proxyPort` (Optional)
-* `org.killbill.billing.plugin.bridge.connectTimeOut` (Optional)
-* `org.killbill.billing.plugin.bridge.readTimeOut` (Optional)
-* `org.killbill.billing.plugin.bridge.requestTimeout` (Optional)
-* `org.killbill.billing.plugin.bridge.strictSSL` (Optional)
-* `org.killbill.billing.plugin.bridge.SSLProtocol` (Optional)
-* `org.killbill.billing.plugin.bridge.internalPaymentMethodIdName` (Default to `internalPaymentMethodId`)
-* `org.killbill.billing.plugin.bridge.controlPlugins` (Optional) 
-* `org.killbill.billing.plugin.bridge.pluginProperties` (Optional)
-
+Here's a configuration example:
+
+```
+!!org.killbill.billing.plugin.bridge.BridgeConfig
```

Comment:
Ok, i think i got the answer to my question [here](https://github.com/killbill/killbill-plugin-framework-java/pull/23#pullrequestreview-110565621)